webapp/src/components/FilterPanel.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from "react";
+
+interface FilterTagProps {
+    tag: string;
+    selected: boolean;
+    onClick: (selected: string) => void;
+}
+
+export class FilterTag extends React.Component<FilterTagProps> {
+    constructor(props: FilterTagProps) {
+        super(props)
+        this.clickHandler = this.clickHandler.bind(this);
+    }
+
+    render() {
+        return <div className="filter-tag">
+            <div className="filter-tag-box" role="checkbox" onClick={this.clickHandler} aria-checked={this.props.selected}>
```

Comment:
```suggestion
            <div className="filter-tag-box" role="checkbox" onClick={this.clickHandler} onKeyDown={sui.fireClickOnEnter} aria-checked={this.props.selected}>
```

don't apply this one as it requires sui import / it'll autoimport it if you type it out, the image editor isn't currently keyboard accessible but still good practice to handle key events to make things easier for future us :) Applies pretty much anywhere you add an `onClick`, it binds enter/space with focus to trigger `onClick`

Reply:
ooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo cool good to know!